bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Target.scala

```diff
@@ -126,13 +125,14 @@ object Target {
   }
 
   object SiderealTarget {
+
+    def apply(name: String, coordinates: Coordinates, magnitudes: List[Magnitude]): SiderealTarget = SiderealTarget(name, coordinates, None, None, None, magnitudes)
```

Comment:
Yeah I just find that default arguments (which is basically what this does) tend to lead to bugs. You might implement it in terms of `SiderealTarget.empty.copy(...)` and then you'll be insulated from the other changes I'm making.